apps/src/netsim/NetSimVisualization.js

```diff
@@ -258,6 +258,44 @@ NetSimVisualization.prototype.getElementByEntityID = function (elementType, enti
 };
 
 /**
+ * Gets the set of VizWires directly attached to the given VizNode on
+ * the local end for which there are also corresponding VizWires coming
+ * from the opposite end. Note that if the VizNode is a router, we
+ * consider all attached wires to be reciprocated.
+ * @param {NetSimVizSimulationNode} vizNode
+ * @returns {Array.<NetSimVizSimulationWire>} the attached wires
+ */
+NetSimVisualization.prototype.getReciprocatedWiresAttachedToNode = function (vizNode) {
+
+  if (vizNode.isRouter) {
+    return this.getWiresAttachedToNode(vizNode);
+  }
+
+  var localWires = this.getLocalWiresAttachedToNode(vizNode);
+
+  return localWires.filter(function (localWire) {
+
+    if (localWire.remoteVizNode.isRouter) return true;
+
+    return this.getWiresAttachedToNode(localWire.remoteVizNode).filter(function (wire) {
+      return wire.remoteVizNode === vizNode;
+    }).length > 0;
+
+  }.bind(this));
```

Comment:
oh, great! The only reason I tend to avoid it is I've encountered it causing confusion when people aren't familiar with that option. I definitely far prefer it, and I'll update my code